src/main/java/com/utils/CreatePropertiesFile.java

```diff
@@ -0,0 +1,41 @@
+package com.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class CreatePropertiesFile {
```

Comment:
is this a method name or a class name what is it? Also if you "create" something where is the create method or create logic at least?